package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "@asbjorn/eslint-plugin-groq",
-  "version": "0.1.2",
+  "version": "0.2.0",
   "description": "Eslint plugin for GROQ queries",
   "author": "Asbjørn Hegdahl",
   "license": "MIT",
+  "engines": {
+    "node": ">=10.0.0"
```

Comment:
I just realized that this is a breaking change. Could you update the version to `1.0.0` instead and put something about node support in the changelog?

Reply:
Figured it would be safe since it was tested on `lts/*` on Travis. `0.x` versions usually are "may break on you" in my experience as well. But sure I can make it 1.0.0 :+1:

Reply:
Done. I also put the Node version explicitly in `.travis.yml` so the tests actually run on the `engine` specified (latest maintenance version of the 10.x branch in this case).

Reply:
Yeah, but we're now moving from "maybe working on node < 10" to "definitely not working on node < 10" :) And I like semver better than that `0.x` stuff anyway